servers/rendering/rendering_device.h

```diff
@@ -50,56 +58,16 @@ class RDPipelineColorBlendState;
 class RDFramebufferPass;
 class RDPipelineSpecializationConstant;
 
-class RenderingDevice : public Object {
+class RenderingDevice : public Object, public RenderingDeviceCommons {
```

Comment:
This is probably not going to work (or not warranted to work) with Godot binding system because depending on the implementation of multiple inheritance, the compiler will do offset adjustments to the pointer when casting, which breaks Godot type system.

This is why I originally suggested doing this different.

Reply:
Alternatively you could make RenderingDeviceCommons inherit from object and simply not registering RenderingDeviceDriver to the binding system (no GDCLASS) that should work fine.

Reply:
I'll do just that in the next push.